web/diff.jsp

```diff
@@ -50,28 +50,15 @@ private String getAnnotateRevision(DiffData data) {
     }
     return "";
 }
-%><%@
-
-include file="mast.jsp"
-
-%><script src="<%=request.getContextPath()%>/js/diff-0.0.1.js" type="text/javascript"></script><%
-/* ---------------------- diff.jsp start --------------------- */
+%>
+<%
 {
     PageConfig cfg = PageConfig.get(request);
     DiffData data = cfg.getDiffData();
-
-    if (data.errorMsg != null)  {
-
-%>
-<div class="src">
-    <h3 class="error">Error:</h3>
-    <p><%= data.errorMsg %></p>
-</div><%
-
-    } else if (data.type == DiffType.TEXT
+    request.setAttribute("diff.jsp-data", data);
+    if (data.type == DiffType.TEXT
             && request.getParameter("action") != null
             && request.getParameter("action").equals("download")) {
-        response.resetBuffer(); // reset buffer to clean it from the header html
         try (OutputStream o = response.getOutputStream()) {
```

Comment:
Yep, I was suggesting to add a comment to the jsp so that no one moves the block of code in the future to prevent reintroducing the bug.